packages/foreman/foreman/foreman.spec

```
@@ -1016,6 +1016,9 @@ exit 0
 %systemd_postun_with_restart %{name}.service
 
 %changelog
+* Mon Jun 22 2020 Avi Sharvit <sharvita@gmail.com> - 2.2.0-0.27.develop
+-  - update foreman-js dependencies
```

nitpick - looks like there is an extra ` - ` here
```suggestion
- update foreman-js dependencies
```